### PR TITLE
Add missing token arbi & base

### PR DESCRIPTION
**Pull request by Drwillean**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Comment by Drwillean** [comment]
I have read the CLA Document and I hereby sign the CLA

File: tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 1]
```diff
@@ -82,4 +82,5 @@ FROM (VALUES
     , (0xD460f305B150Bc70b063E806815B50BebE9F21b1, 'APRTo', 18)
     , (0x577fd586c9e6ba7f2e85e025d5824dbe19896656, 'SYNO', 18)
     , (0x6985884c4392d348587b19cb9eaaf157f13271cd, 'ZRO', 18)
+    , (0xa7997f0ec9fa54e89659229fb26537b6a725b798, 'PAL', 18)
```
Comment (Hosuke) [inline tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql:85]: https://arbiscan.io/token/0xa7997f0ec9fa54e89659229fb26537b6a725b798

File: tokens/models/tokens/base/tokens_base_erc20.sql [hunk 2]
```diff
@@ -239,4 +239,5 @@ FROM (VALUES
     , (0x96419929d7949D6A801A6909c145C8EEf6A40431, 'SPEC', 18)  
     , (0x7ab8ea5ef610f979e1c59fbd9da9284672e5a136, 'KABOSU', 18)  
     , (0xd722e55c1d9d9fa0021a5215cbb904b92b3dc5d4, 'RDNT', 18)  
+    , (0x224114c444e3c3d936532fe08351648700570909, 'PAL', 18)
```
Comment (Hosuke) [inline tokens/models/tokens/base/tokens_base_erc20.sql:242]: https://basescan.org/token/0x224114c444e3c3d936532fe08351648700570909

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @Drwillean

**Pull request merged**